src/app.ts

```diff
@@ -80,7 +80,10 @@ export function createHandle (stack: Stack): PHandle {
       }
     }
     if (!res.writableEnded) {
-      throw createError(404, 'Not Found')
+      throw createError({
+        statusCode: 404,
+        statusMessage: 'Not Found'
```

Comment:
I would prefer to see `message` instead.
I would also vote for `status` instead of `statusCode` 😄

Reply:
But it can makes sense to keep them like this, cc @pi0

Reply:
The idea is to be consistent with the http response object but `status` and `message` is much simpler

Reply:
Indeed. Idea of h2 is keeping framework as much as consistent with node Http (https://nodejs.org/api/http.html#http_response_statuscode)